graphicsmagick.yaml

```diff
@@ -20,25 +20,43 @@ environment:
       - libtool
       - libwebp-dev
       - libxml2-dev
+      - mercurial
       - tiff-dev
 
 pipeline:
-  - uses: fetch
-    with:
-      expected-sha256: 2b88580732cd7e409d9e22c6116238bef4ae06fcda11451bf33d259f9cbf399f
-      uri: https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/${{package.version}}/GraphicsMagick-${{package.version}}.tar.xz
+  - runs: |
+      # Clone the repository.
```

Comment:
We should update these comments to say that the commit below needs to be updated to match the release

Reply:
Great heads up, thanks RJ, did an attempt to add it